src/app/components/game-center/game-center.component.ts

```diff
@@ -33,12 +33,9 @@ export class GameCenterComponent implements OnInit {
     this.toastr.warning('Palabra no encontrada');
   }
 
-  notSearch() {
-    this.encontrado = false;
-  }
 
-  setSearched(valor: boolean) {
-    this.encontrado = valor;
+  isSearched(){
+    return this.found;
```

Comment:
```suggestion
```
U don't need this function, if u want to check the boolean, just get the variable `found`